README.md

```diff
@@ -1,15 +1,12 @@
 # Ember FastBoot
 
-An Ember CLI addon that allows you to render and serve Ember.js apps on
-the server. Using FastBoot, you can serve rendered HTML to browsers and
-other clients without requiring them to download JavaScript assets.
+An Ember CLI Addon that allows you to render and serve Ember.js apps on the server. Using FastBoot, you can serve rendered HTML to browsers and other clients without requiring them to download JavaScript assets.
 
-Currently, the set of Ember applications supported is extremely limited.
-As we fix more issues, we expect that set to grow rapidly. See [Known
-Limitations](#known-limitations) below for a full-list.
+## How It Works
 
-The bottom line is that you should not (yet) expect to install this add-on in
-your production app and have FastBoot work.
+FastBoot works by creating a Node virtual [sandbox](https://github.com/ember-fastboot/fastboot/blob/master/lib/ember-app.js#L50-L75) and [executing](https://github.com/ember-fastboot/fastboot/blob/master/lib/ember-app.js#L165-L232) your Ember Application within it.
+
+Most Ember applications should work out of the box. However, there are some idiomatic patterns of design that you have to be sure to follow. See [Known Limitations](#known-limitations) below for a full-list.
```

Comment:
I think maybe changing this to:

> Most Ember applications should work out of the box. However, there are some patterns that you will make sure your app runs with FastBoot. See [Known Limitations](#known-limitations) below for a full-list.

I would stay clear of `idiomatic` to make things less intimidating.
Especially in the docs/readme.